google/cloud/aiplatform/models.py

```diff
@@ -840,6 +853,13 @@ def _deploy(
                 be immediately returned and synced when the Future has completed.
             deploy_request_timeout (float):
                 Optional. The timeout for the deploy request in seconds.
+            autoscaling_target_cpu_utilization (int):
+                Target CPU Utilization to use for Autoscaling Replicas.
```

Comment:
Could you add "Optional" to the beginning of this docstring? Same for `autoscaling_target_accelerator_duty_cycle`.

Reply:
Fixed in https://github.com/googleapis/python-aiplatform/pull/1197/commits/7a601c7266c9e871c5607b6375bfbbfc28be992b